gradle/versions.gradle

```
@@ -13,8 +13,8 @@
 version = '0.22.0-SNAPSHOT'
 
 ext.versions = [
-	'xtend_lib': '2.28.0',
-	'guava': '[30.1,31)',
+	'xtend_lib': '2.23.0.M2',

```

am not sure how to properly chickend and egg bootstrap this? use range?

---

Is this a typo? Why is this version older?

---

I'm guess a typo as 2.**32**.0.M2 was [published a few days ago](https://central.sonatype.com/artifact/org.eclipse.xtend/org.eclipse.xtend.core/2.32.0.M2)

A bit concerning that the build can succeed without this being checked.

---

> A bit concerning that the build can succeed without this being checked.

(Sorry for stream of consciousness as I learn what is going on)

It probably succeeded because there is a 2.23.0.M2 in maven central too.

```suggestion
	'xtend_lib': '2.32.0.M2',

```